feature_layers/change-feature-layer-renderer/README.md

```diff
@@ -1,27 +1,34 @@
-# Change Feature Layer Renderer
+# Change feature layer renderer
 
-Change how a feature layer looks with a renderer.
+Change the appearance of a feature layer with a renderer.
 
-![](ChangeFeatureLayerRenderer.gif)
+![Image of change feature layer renderer](ChangeFeatureLayerRenderer.png)
+
+## Use case
+
+A feature layer hosted on ArcGIS Online has a preset renderer and will display in an ArcGIS Runtime application with that renderer. However, for example, the color associated with the original renderer may be unsuitable for a company with staff or clients who are color blind, or for presentation with a different set of basemap and operational layers. In these cases, the renderer on the feature layer's data can be set to a more suitable color.
 
 ## How to use the sample
 
-Use the buttons in the control panel to change the renderer.
+Use the button to change the renderer on the feature layer. The original renderer displays orange circles, the diameters of which are proportional to carbon storage of each tree. When the blue renderer in this sample is applied, it displays the location of the trees simply as blue points.
```

Comment:
with the recent changes, there is no blue renderer: it switches between being green dots and orange circles. So we should change the sample to be blue rather than green me thinks